### PR TITLE
HEMTT Build System

### PR DESCRIPTION
**Pull request by DerZade**
This PR will introduce automatic building of all PBOs via [HEMTT](https://github.com/SynixeBrett/HEMTT).

### CODE REVIEW

File: .github/workflows/add-to-release.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: 'Add to Release'
+
+on: [release]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (TheWillard) [inline .github/workflows/add-to-release.yml]: WIndows pls

Comment (TheWillard) [inline .github/workflows/add-to-release.yml]: Windows pls

File: .github/workflows/buildMod.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: '[CI] Build Mod'
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (TheWillard) [inline .github/workflows/buildMod.yml]: Windows pls

File: .github/workflows/add-to-release.yml (hunk 1 above)
Comment (DerZade) [reply .github/workflows/add-to-release.yml]: Windows won't work, as you can see here: https://github.com/gruppe-adler/grad_meh/pull/2/commits/6b6514dc265f941c29b2d42f7e59c1086e4c785e

File: .github/workflows/buildMod.yml (hunk 2 above)
Comment (DerZade) [reply .github/workflows/buildMod.yml]: Windows won't work, as you can see here: https://github.com/gruppe-adler/grad_meh/pull/2/commits/6b6514dc265f941c29b2d42f7e59c1086e4c785e

**Pull request merged**